src/app/login/FormComponent.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import { Input, ButtonPrimary, FormItem } from "@/components/atoms";
+import * as Yup from "yup";
+import { ILogIn } from "@/types/auth";
+import { useRouter } from 'next/navigation'
+import Auth from "@/service/auth.service";
+import Link from 'next/link';
+
+const FormComponent = () => {
+  const [loading, setLoading] = useState(false)
+  const router = useRouter();
+  const SignUpValidationSchema = Yup.object().shape({
+    email: Yup.string().email("Email inválido").required("Campo email é requerido"),
+    password: Yup.string()
+      .matches(
+        /(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]/,
+        "Deve conter pelo menos uma letra e um número"
+      )
+      .required(
+        "Coloque uma combinação de numeros, letras e sinais de pontuação (como ! e &)."
+      )
+      .min(8, "Senha deve conter no mínimo 8 caracteres.")
+      .max(36, "Senha não deve contar mais de 36 caracteres"),
+  });
+
+  const initialValues: ILogIn = {
+    email: "",
+    password: "",
+  };
+
+  const handleFormSubmit = async (values: ILogIn) => {
+    setLoading(true);
+    await Auth.login({
+      email: values.email,
+      password: values.password,
+      }).then(()=>{
```

Comment:
```suggestion
      }).then((response)=>{
```

because u need know what is the backend response.